runtime_data/postprocess/gb.patterns

```diff
@@ -1 +1,33 @@
-gb	@@##@@@
+gb	[A-HK-PR-TV-Y][A-HJ-PRSTV-Y]##[A-HJ-PR-Z][A-HJ-PR-Z][A-HJ-PR-Z]
+
+gb	[A-HJ-NP-TV-Y]#[A-HJ-PRSTV-Y][A-HJ-PRSTV-Y][A-HJ-PRSTV-Y]
+gb	[A-HJ-NP-TV-Y]##[A-HJ-PRSTV-Y][A-HJ-PRSTV-Y][A-HJ-PRSTV-Y]
+gb	[A-HJ-NP-TV-Y]###[A-HJ-PRSTV-Y][A-HJ-PRSTV-Y][A-HJ-PRSTV-Y]
+
+gb	[A-HJ-PRSTV-Y][A-HJ-PRSTV-Y][A-HJ-PRSTV-Y]#[A-HJ-NP-TV-Y]
+gb	[A-HJ-PRSTV-Y][A-HJ-PRSTV-Y][A-HJ-PRSTV-Y]#[A-HJ-NP-TV-Y]
+gb	[A-HJ-PRSTV-Y][A-HJ-PRSTV-Y][A-HJ-PRSTV-Y]#[A-HJ-NP-TV-Y]
+
+gb	###[A-HJ-PR-Y][A-HJ-PR-Y][A-HJ-PR-Y]
+gb	##[A-HJ-PR-Y][A-HJ-PR-Y][A-HJ-PR-Y]
+gb	#[A-HJ-PR-Y][A-HJ-PR-Y][A-HJ-PR-Y]
+gb	####[A-HJ-PR-Y][A-HJ-PR-Y]
+gb	###[A-HJ-PR-Y][A-HJ-PR-Y]
+gb	##[A-HJ-PR-Y][A-HJ-PR-Y]
+gb	#[A-HJ-PR-Y][A-HJ-PR-Y]
+gb	####[A-HJ-PR-Y]
+gb	###[A-HJ-PR-Y]
+gb	##[A-HJ-PR-Y]
+gb	#[A-HJ-PR-Y]
+gb	[A-HJ-PR-Y][A-HJ-PR-Y]####
+gb	[A-HJ-PR-Y][A-HJ-PR-Y]###
+gb	[A-HJ-PR-Y][A-HJ-PR-Y]##
+gb	[A-HJ-PR-Y][A-HJ-PR-Y]#
+gb	[A-HJ-PR-Y]####
+gb	[A-HJ-PR-Y]###
+gb	[A-HJ-PR-Y]##
+gb	[A-HJ-PR-Y]#
+gb	[A-PR-Y][A-PR-Y][A-PR-Z]#### 
```

Comment:
There is also the 'QNI' special plates which can have 1-4 numbers

Reply:
Added these in commit afe23c3c424247006c1e48f1145be8a86db6a271